apps/console/src/features/identity-providers/components/settings/connected-apps.tsx

```diff
@@ -0,0 +1,529 @@
+                key: "name",
+                render: (app: ApplicationListItemInterface): ReactNode => {
+
+                    /**
+                     * Note: the templateId for Standard-Based Applications in applicationTemplates is
+                     * 'custom-application'(only 1 template is available).
+                     * But backend passes 3 distinct ids for Standard Based Applications.
+                     */
+
+                    // Create a set with custom-application Ids.
+                    const customApplicationIds: Set<string> = new Set([
+                        ApplicationManagementConstants.CUSTOM_APPLICATION_SAML,
+                        ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC,
+                        ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
+                    ]);
+
+                    let templateDisplayName: string = "";
+
+                    // Checking whether the templateId from backend, is for a custom application.
+                    if (customApplicationIds.has(app.templateId)) {
+                        templateDisplayName = applicationTemplates
+                            && applicationTemplates instanceof Array
+                            && applicationTemplates.length > 0
+                            && applicationTemplates.find((template) => {
+                                return template.id === ApplicationManagementConstants.CUSTOM_APPLICATION;
+                            }).name;
+                    } else {
+                        const relevantApplicationTemplate: ApplicationTemplateListItemInterface | undefined = 
+                            applicationTemplates
+                            && applicationTemplates instanceof Array
+                            && applicationTemplates.length > 0
+                            && applicationTemplates.find((template) => {
+                                return template.id === app.templateId;
+                            });
+
+                        if (relevantApplicationTemplate?.templateGroup) {
+                            const templateGroupId: string = relevantApplicationTemplate.templateGroup;
+
+                            templateDisplayName = groupedApplicationTemplates
+                                && groupedApplicationTemplates instanceof Array
+                                && groupedApplicationTemplates.length > 0
+                                && groupedApplicationTemplates.find((group) => {
+                                    return (group.id === templateGroupId || group.templateGroup === templateGroupId);
+                                }).name;
+                        }
+                    }
+
+                    return (
+                        <Header
+                            image
+                            as="h6"
+                            className="header-with-icon"
+                            data-testid={ `${ testId }-item-heading` }
```

Comment:
Let's use `componentid` here as well. Check in other places

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3494/commits/05d52641ee34bd16b1f267ba20fb05d9c9c627bf